packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -0,0 +1,176 @@
+  const entityId = SynapseUtilityFunctions.parseEntityIdFromSqlStatement(sql)
+  const queryBundleRequest: QueryBundleRequest = {
+    partMask:
+      SynapseConstants.BUNDLE_MASK_QUERY_SELECT_COLUMNS |
+      SynapseConstants.BUNDLE_MASK_QUERY_RESULTS,
+    concreteType: 'org.sagebionetworks.repo.model.table.QueryBundleRequest',
+    entityId,
+    query: {
+      sql,
+    },
+  }
+  const { data: queryResultBundle } =
+    useGetQueryResultBundle(queryBundleRequest)
+
+  const dataRows = queryResultBundle?.queryResult!.queryResults.rows ?? []
+
+  enum ExpectedColumns {
+    TAG = 'Tag',
+    JOURNAL = 'Journal',
+    TITLE = 'Title',
+    PUBLICATION_DATE = 'publicationDate',
+  }
+
+  const tagColIndex = getFieldIndex(ExpectedColumns.TAG, queryResultBundle)
+  const journalColIndex = getFieldIndex(
+    ExpectedColumns.JOURNAL,
+    queryResultBundle,
+  )
+  const titleColIndex = getFieldIndex(ExpectedColumns.TITLE, queryResultBundle)
+  const publicationDateColIndex = getFieldIndex(
+    ExpectedColumns.PUBLICATION_DATE,
+    queryResultBundle,
+  )
+
+  return (
+    <Box
+      sx={{
+        display: 'flex',
+        flexDirection: isSmallScreen || isMediumScreen ? 'column' : 'row',
+        gap: isSmallScreen || isMediumScreen ? '38px' : '80px',
+      }}
+    >
+      <Grid
+        container
+        spacing={2}
+        sx={{
+          order: isSmallScreen || isMediumScreen ? 1 : 0,
+          flex: isSmallScreen || isMediumScreen ? '0 0 100%' : '3',
+        }}
+      >
+        {dataRows.map(pub => (
+          <Grid item xs={12} sm={6} md={4} key={pub.rowId}>
+            <Card sx={{ height: '100%', boxShadow: 'none', border: 'none' }}>
+              <CardContent>
+                <Box display={'flex'} gap={'6px'} flexWrap={'wrap'}>
+                  {pub.values[tagColIndex] &&
+                    JSON.parse(pub.values[tagColIndex] as string).map(
+                      (tag: string, index: number) => (
+                        <Typography
+                          key={index}
+                          variant="overline"
+                          sx={{
+                            backgroundColor: '#EAECEE',
+                            borderRadius: '3px',
+                            padding: '4px 8px',
+                            border: 'none',
+                            lineHeight: 'initial',
+                          }}
+                        >
+                          {tag}
+                        </Typography>
+                      ),
+                    )}
+                </Box>
+                <Typography
+                  variant="headline2"
+                  sx={{
+                    padding: '20px 0px',
+                  }}
+                >
+                  {pub.values[titleColIndex]}
+                </Typography>
+                <Typography
+                  variant="body1"
+                  color="#71767F"
```

Comment:
It looks like at some point our `grey.700` diverged from the [style library](https://www.figma.com/design/0oPm5lLSUva8kyfVNMS6FA/Sage-Style-%26-Component-Library?node-id=77-1232&node-type=canvas&m=dev).

Can you update `packages/synapse-react-client/src/theme/palette/Palettes.ts` so `grey.700` is `#71767F`?

```suggestion
                  color="grey.700"
```